.github/workflows/test.yml

```diff
@@ -132,22 +132,22 @@ jobs:
   #         path: debug_files.tar.gz
   #         if-no-files-found: ignore
 
-  upload-cache:
-    if: github.event_name == 'push'
-    needs: ["integration_tests"]
-    strategy:
-      matrix:
-        os: [macos-latest]
-    runs-on: ${{ matrix.os }}
-    steps:
-      - uses: actions/checkout@v2
-      - uses: cachix/install-nix-action@v18
-      - uses: cachix/cachix-action@v11
-        with:
-          name: ethermint
-          signingKey: "${{ secrets.CACHIX_SIGNING_KEY }}"
-      - name: 'instantiate integration test env'
-        run: nix-store -r $(nix-instantiate tests/integration_tests/shell.nix)
+  # upload-cache:
```

Comment:
The upload-cache test needs the `integration test` which has an issue with nixOS because of the added gql dependency as stated in a previous PR.
Commented out the upload-cache test as all the tests in test.yml file were being ignored because of this dependency error.